pkgs/applications/misc/octoprint/plugins.nix

```diff
@@ -289,6 +435,27 @@ in
     };
   };
 
+  resource-monitor = buildPlugin rec {
+    pname = "resource-monitor";
+    version = "0.3.16";
+
+    src = fetchFromGitHub {
+      owner = "Renaud11232";
+      repo = "OctoPrint-Resource-Monitor";
+      rev = version;
+      sha256 = "sha256-w1PBxO+Qf7cSSNocu7BiulZE7kesSa+LGV3uJlmd0ao=";
+    };
+
+    propagatedBuildInputs = with super; [ psutil  ];
+
+    meta = with lib; {
+      description = "A plugin to view the current CPU and RAM usage on your system";
```

Comment:
```suggestion
      description = "Plugin to view the current CPU and RAM usage on your system";
```

Reply:
Would like to keep it consistent with the other plugins, as well as keeping it the same as the official plugin's repo